docker/grade.sh.template

```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
+
+################################################################################
+## This shell script wraps the sbt invocation to set the arguments up         ##
+## properly, and redirects the noisy stdout to stderr, and stderr to stdout   ##
+## so that Coursera's grading systems can pick up the grades and feedback.    ##
+##                                                                            ##
+## Important security assumption: nothing except the grade output must be     ##
+## allowed to go to stderr, especially nothing controlled by the submission.  ##
+##                                                                            ##
+## Initial Author: Brennan Saeta <saeta@coursera.org>                         ##
+## For background / further details, please see:                              ##
+##    https://github.com/sbt-coursera/sbt-coursera                            ##
+################################################################################
+
+# Initialize the part id with a default value
+PART_ID="02Vi5q7m"
+
+# Parse command line parameters looking for parameter partId
+while [[ $# > 1 ]]
+do
+  key="$1"
+  case $key in
+    partId|--partId)
+      PART_ID="$2"
+      shift
+      ;;
+    *)
+      # Unknown key-value pair
+      shift
+      ;;
+  esac
+  shift
+done
+
+cd /grader
+
+SET_CMD="set(partIdOfGradingProject in submissionProject) := \"$PART_ID\""
+2>&1 echo Running: sbt "'$SET_CMD'" submission/grade
+sbt "'$SET_CMD'" submission/grade 3>&2 2>&1 1>&3
```

Comment:
I think it might be best if I can talk through the security assumptions of GrID over the phone call.